src/assessment/migrations/0001_initial.py

```diff
@@ -0,0 +1,37 @@
+# Generated by Django 5.1.5 on 2025-02-05 08:40
```

Comment:
@avishkar-sapkal Instead of the multiple file make sure you create only one migration file for first version